src/Pages/PlayList/PlayList.js

```diff
@@ -0,0 +1,88 @@
+import axios from 'axios';
+import React from 'react'
+import { useEffect } from 'react';
+import { usePlayList } from '../../Context/PlayListContext';
+// import { Navigate } from 'react-router-dom';
+const authtoken = localStorage.getItem("AuthToken");
+
+export const PlayList = () => {
+
+    const {playListState} = usePlayList();
+    const {playListDispatch,deletePlayList} = usePlayList();
+
+    console.log(playListState,"9090")
+
+    useEffect(() => {
+      (async () => {
+        try {
+          const response = await axios.get("/api/user/playlists", {
+            headers: {
+              authorization:authtoken,
+            },
+          });
+  
+          console.log("response", response.data.playlists);
+          playListDispatch({
+            type: "ADD_TO_PlayList",
+            payload: response.data.playlists,
+          });
+        } catch (error) {
+          console.log(error);
```

Comment:
you can use `console.error` for err handling

Reply:
ok